src/lib/mongoBackend/safeMongo.cpp

```diff
@@ -69,8 +108,8 @@ BSONObj getObjectField(const BSONObj& b, const std::string& field, const std::st
   }
   else
   {
-    LM_E(("Runtime Error (field '%s' was supposed to be an object but type=%d in BSONObj <%s> from caller %s:%d)",
-          field.c_str(), b.getField(field).type(), b.toString().c_str(), caller.c_str(), line));
+    LM_E(("Runtime Error (field '%s' was supposed to be an object but the type is '%s' in BSONObj <%s> from caller %s:%d)",
```

Comment:
Maybe including both number and string could be useful (general comment for all similar LM_E in this PR).

Reply:
Fixed in a794057